src/lib/support/Variant.h

```diff
@@ -70,6 +70,15 @@ struct VariantCurry<>
     inline static void Copy(std::size_t that_t, const void * that_v, void * this_v) {}
 };
 
+// Using assert inside template will encoded template name into the assert
+// error message, which cost lots of flash space, this function can strip the
+// type name, and the function name should be as short as possible to save
+// space.
+inline void AssertStrip(bool exp)
+{
+    assert(exp);
```

Comment:
We shouldn't be using `assert` at all, last I checked, instead of playing this game around stripping out the function name...  Maybe a followup on replacing this `assert` with something we _should_ be using?